core/services/vrf/listener_v2.go

```diff
@@ -290,12 +317,13 @@ func (lsn *listenerV2) processRequestsPerSub(
 	startBalance *big.Int,
 	maxGasPriceWei *big.Int,
 	reqs []pendingRequest,
-) {
+) map[string]struct{} {
+	var processed = make(map[string]struct{})
```

Comment:
Do you think it's a good idea to also add some timing measurement local to this method, like you did for `processPendingVRFRequests`?